packages/salesforcedx-vscode-apex/README.md

```diff
@@ -19,19 +19,19 @@ To see code-completion suggestions, press Ctrl+space when you’re working in a
 
 ## View or Jump to Definitions
```

Comment:
@ruthemmanuelle @ekapner - I wonder if it would be useful to have a new section before this one to explicitly call out that View or Jump to Definition and Find All References currently work for only user-defined types. When you use it on types/methods/fields from Apex Standard Library or Managed Packages, nothing is returned.

Reply:
@vazexqi - I think that's pretty clear from the language in the existing sections. It seems weird to me to add an extra section to duplicate this information. If I were scanning the README, I think I'd be more likely to notice "user-defined" in the list of stuff that works rather than in a separate section. Do you think the existing language (after this PR) is unclear or inaccurate?

Reply:
@ruthemmanuelle - What you have written is accurate – it describes what **works** currently. I was just curious if we should call out what does **not** work (basically known limitations). 

I think we should leave this as-is. If we get questions/issues from this we can consider whether to add that explicitly.